### DISCUSSION TITLE
Daily 24/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#156] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Dificultades o Impedimentos.

**Comment by BraianTroncoso** [comment]
Ayer me tomé el día y no toque nada con respecto a la tec.
Hoy estoy haciendo unos flyers para un trabajo de un amigo y despues me pongo a seguir viendo el curso de Angular.
Ya tengo un par de ideas de como hacer el front.

**Comment by Brian-Almada** [comment]
Hoy avanzando con Angular y pensando qué tareas tomar.

**Comment by rpgrca** [comment]
Otro día donde hice bastante poco por tener que ir a Capital y después trabajar, sin embargo voy a empezar con al menos la pantalla de contacto como para practicar.

**Comment by NataliaAlvarezIspc** [comment]
Entre el viernes y el fin de semana estuve estudiando y viendo videos sobre angular. El domingo tuvimos la reunión por meet de la review y planning.
Hoy tuve que ocuparme de temas personales y medicos, así que no le pude dedicar tiempo al proyecto.
